src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  close: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  close,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(todo.userId).then(setUser);
```

Comment:
You have not handled the case if 'getUser' function fails. It would be beneficial to handle any potential errors that might arise from the API call to provide a better user experience.